examples/algorithms/cpu/include/traccc/clusterization/clusterization_algorithm.hpp

```diff
@@ -66,9 +66,10 @@ class clusterization_algorithm
             auto module = cells_per_event.at(i).header;
 
             // The algorithmic code part: start
-            traccc::cluster_collection clusters_per_module = cc->operator()(
+            traccc::host_cluster_container clusters_per_module = cc->operator()(
```

Comment:
You should rename the variable as well.

In the "new EDM" we don't store the clusters separately for the modules. So the variable name should not try to confuse people about it. :wink:

Reply:
alright, done.